demisto_sdk/commands/common/tools.py

```diff
@@ -758,36 +766,39 @@ def get_last_remote_release_version():
                 f'{exc_msg[exc_msg.find(">") + 3:-3]}.\n'
                 f"This may happen if you are not connected to the internet."
             )
-        logger.info(
-            f"[yellow]Could not get latest demisto-sdk version.\nEncountered error: {exc_msg}[/yellow]"
+        logger.warning(
+            f"Could not find the latest version of 'demisto-sdk'.\nError: {exc_msg}"
```

Comment:
why remove `yellow` as part of this PR?

Reply:
Using a `warning` log level already uses a yellow color automatically, I already checked and confirmed it.
These are just some remains from when we didn't do that automatically and then these tags were needed.